### ISSUE TITLE
Hyperparameters for replicating supervised MT ro -> en result.

### ISSUE THREAD

**Issue by sarthakgarg** [facebookresearch/XLM#37]
Hi,

I am trying to replicate the supervised MT ro -> en baseline of 28.4 mentioned in the paper. I was hoping that you could give me some idea about the hyperparameters for that.
Specifically can you tell me the values of #of BPE operations, learning rate and learning rate schedule used, dropout and attention dropout values, embedding size of the network, batch size and # of gpus used during training.

Thanks!

**Comment by glample** [comment]
Hi, we used the parameters of the README https://github.com/facebookresearch/XLM#train-on-unsupervised-mt-from-a-pretrained-model , with 60000 BPE operations, and 8 GPUs.

**Comment by sarthakgarg** [comment]
Thanks for that! I figured out the issue. I was only using EuroParl v8 parallel corpus and not the full (Europarl v8 + SETIMES2) wmt16 dataset.
